assets/stylesheets/search.css

```
@@ -82,6 +82,11 @@ ol.search-snippets li {
   padding: 0.25em 0;
 }
 
+.last-modified::after {
```

Could you limit this style only in search result something like the following?

```suggestion
#search-result .last-modified::after {
```

---

fix: [b66f956](https://github.com/clear-code/redmine_full_text_search/pull/119/commits/b66f956fdc28d8b42270367302c821bb20d352fa)